scripts/wdl/case_gatk_acnv_workflow.wdl

```diff
@@ -319,7 +340,50 @@ task AllelicCNV {
 
     output {
         File acnv_final_segs="${entity_id}-sim-final.seg"
-        File acnv_final_segs_gatk_compatible="${entity_id}-sim-final.cnv.seg"
-        File acnv_final_segs_acs_compatible="${entity_id}-sim-final.acs.seg"
+        #File acnv_final_segs_gatk_compatible="${entity_id}-sim-final.cnv.seg"
+        #File acnv_final_segs_acs_compatible="${entity_id}-sim-final.acs.seg"
+    }
+}
+
+
+#Create plots of copy number variant data
+task PlotSegmentedCopyRatio { 
+    File jar_file
+    File tn_file
+    File pre_tn_file
+    File called_file
+    String output_dir
+    Int mem
+
+    command {
+        mkdir -p ${output_dir} && \
+	java -Xmx${mem}g -jar ${jar_file} PlotSegmentedCopyRatio --tangentNormalized ${tn_file} \
+	 --preTangentNormalized ${pre_tn_file} --segments ${called_file} \
+	 --output ${output_dir}
+    }
+
+    output {
+        #corresponding output images
```

Comment:
Output should list `. . ._Full_Genome.png` and `. . ._Before_and_After.png`, but you can skip the other output like `. . .QC.txt`.